common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```
@@ -89,7 +89,7 @@ cACTION_APMANAGER_HOSTAP_ACS_ERROR_NOTIFICATION:
 cACTION_APMANAGER_HOSTAP_ACS_NOTIFICATION:
   _type: class
   cs_params: sApChannelSwitch
-  supported_channels_list:
+  preferred_channels_list:
     _type: beerocks::message::sWifiChannel
     _length: [ "beerocks::message::SUPPORTED_CHANNELS_LENGTH" ]
```

This should also be changed (nitpick)
```suggestion
    _length: [ "beerocks::message::PREFERRED_CHANNELS_LENGTH" ]
```